### PR TITLE
fix: merge kubeconfigs on windows

### PR DESCRIPTION
**Pull request by sralloza**
## Description
Merging kubeconfigs does not work on Windows. This PR fixes it.

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md)) 

Described in https://github.com/alexellis/k3sup/issues/356.

## How Has This Been Tested?
Local execution on windows 10.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
You definitely cannot use syntax like `~/` on Windows cmd.exe

You should be able to use it on Git Bash.

For Windows users, you'd want something more like `%USERPROFILE%`

https://superuser.com/questions/168714/is-there-a-shortcut-command-in-windows-command-prompt-to-get-to-the-current-user

**Comment by sralloza** [comment]
I understand It's unusual to use `~/` in windows, but with only one line of code it can be supported here.

What about the other issues?

**Comment by alexellis** [comment]
In Go, all errors should have a lowercase first letter

I noticed you have 4 commits, it'd be better if you could rebase these into one.

File: cmd/install.go [hunk 1]
```diff
@@ -420,6 +425,11 @@ func mergeConfigs(localKubeconfigPath, context string, k3sconfig []byte) ([]byte
 		return nil, fmt.Errorf("Could not merge kubeconfigs: %s", err)
 	}
 
+	err = file.Close()
+	if err != nil {
+		return nil, errors.Wrapf(err, "Could not close temporary kubeconfig file: %s", file.Name())
```
Comment (alexellis) [inline cmd/install.go:430]: Better to use: `fmt.Errorf("message... %w", err)`

**Pull request closed without merging**